plugins/modules/azure_rm_aks.py

```diff
@@ -870,6 +908,14 @@ def __init__(self):
                 type='dict',
                 options=service_principal_spec
             ),
+            identity=dict(
+                type='dict',
+                options=managed_identity_spec,
+                required_if = [
```

Comment:
unexpected spaces around keyword / parameter equals
```suggestion
                required_if=[
```